data/40-gdm.rules

```diff
@@ -0,0 +1,8 @@
+polkit.addRule(function(action, subject) {
```

Comment:
This file could be shipped in gdm itself, and then you could substitute the actual username in it, instead of having to special-case Debian-gdm.